setup-environment-internal

```diff
@@ -275,6 +275,29 @@ DEPLOY_DIR = "${BUILDDIR}/deploy"
 # Go through the Firewall
 #HTTP_PROXY = "http://${PROXYHOST}:${PROXYPORT}/"
 _EOF
+
+if [ -d "${HOME}/sstate-cache-mirrors" ]; then
```

Comment:
Should we define a namespace for the OE/Yocto/LmP content?

Something like ${HOME}/yocto or similar?

Reply:
It's a good idea, so as not to spread things in the HOME folder but don't know what name to use for those you suggested

Reply:
I think a clear and generic name first, like yocto, and then a sub folder like lmp would be good enough already.

`${HOME}/yocto/lmp`

That would be easy for folks to understand the content associated with it.

Reply:
I will do that and maybe using a variable so the user can change the default locations

```
# default location
LMP_CACHE_MIRRORS="${YOCTO_CACHE_MIRRORS:-${HOME}/yocto/lmp}"

# mirror
SSTATE_MIRRORS += " \
    file://.* file://${LMP_CACHE_MIRRORS}/sstate-cache-mirrors/PATH \
    "
```

Reply:
@ricardosalveti done with two environment variable to customize the path if needed

- LMP_LOCAL_SSTATE_MIRRORS
- LMP_LOCAL_PRE_MIRRORS